### DISCUSSION TITLE
cn/2024/09/song-of-an-ordinary-man/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#45] [Comments]
# cn/2024/09/song-of-an-ordinary-man/

最近在没有快进的情况下看完了一部剧《凡人歌》，你说由多好看吗，也不是，只是共鸣多些，剧中的种种都好似在“点”我。这一度让我陷入了第二次职业焦虑，但比第一次职业焦虑（大概 5 年前）会显得没那么严重，没那么慌张。

https://leovan.me/cn/2024/09/song-of-an-ordinary-man/

**Comment by auravan** [comment]
我也是凡人，多谢您提点，最近我的网站也上线了，auravan.top。希望我的文章可以给您的生活添一些慰藉。

**Comment by cuixiping** [comment]
巧合的机会发现你的网站诸葛实况 https://zhuge.live/ 是同款风格，不知道你和这个博主是否认识？
本来想在诸葛实况留言的，但是发现他的disqus加载不出来，因为他的页面代码里面还是你的网站的参数。

**Reply by leovan** [reply]
不认识，网站的主题被我搞得越来越复杂了，估计大家使用的时候有些配置没删干净

**Comment by jiu-wei** [comment]
站主写作水平很高，文笔也很细腻。我看了很多篇，这一篇我觉得写的超级棒。第一段写的很像我自己，总是会考虑最坏的情况，但是同时也觉得如果我思考的够全面，那么我就可以规避这些风险和可能出现的坏结果，也很自信的觉得能规避大部分的坏情况。我23年从大学毕业，学习的是软件工程，现在在一家小公司当一个简单的开发，希望以后可以跳槽去更大的厂，看更大的世界。我也有一辆自己的小摩托，但是目前我只能让他带我上下班，有机会的话我也想去自己摩旅一趟。我在偶然之间看到了站主的博客，真的很厉害，站主的生活真的就是我对未来的期望，技术大佬+爱自由爱生活的一个人，希望站主越来越好，同时也希望我能实现自己想要的生活，虽然目前还是有着很大的差距，但以后的事情谁说的准呢，哈哈哈哈哈。希望我们都能越来越好，加油加油！！💪

**Reply by leovan** [reply]
谢谢，一起加油，祝好。
